core/src/main/scala/kafka/tools/ReplicaVerificationTool.scala

```diff
@@ -87,7 +98,7 @@ object ReplicaVerificationTool extends Logging {
                          .defaultsTo(".*")
     val initialOffsetTimeOpt = parser.accepts("time", "Timestamp for getting the initial offsets.")
                            .withRequiredArg
-                           .describedAs("timestamp/-1(latest)/-2(earliest)")
+                           .describedAs("-1(latest)/-2(earliest)")
```

Comment:
To simply code, I have removed the support to start from a particular timestamp offset. Not sure any one using this. If required I will add.

Reply:
If it's not too complicated, we should probably just add it.

Reply:
Updated the code.